src/createServer.js

```diff
@@ -1,3 +1,35 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const validation = require('./validation');
+
+module.exports.createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const text = normalizedURL.pathname.replace('/', '');
+    const caseName = normalizedURL.searchParams.get('toCase');
+
+    res.setHeader('Content-type', 'application/json');
+
+    const errors = validation(text, caseName);
+
+    if (errors.length) {
+      res.status = 400;
+      res.end(JSON.stringify({ errors }));
+
+      return;
+    }
+
+    const converted = convertToCase(text, caseName.toUpperCase());
```

Comment:
The `convertToCase` function is expected to handle case conversion but here it is receiving an already uppercased `caseName`. This might not be correct depending on the implementation of `convertToCase`. Ensure that `convertToCase` expects an uppercased case name, or pass the original `caseName` to it.